webextensions/api/scripting.json

```diff
@@ -0,0 +1,393 @@
+{
+  "webextensions": {
+    "api": {
+      "scripting": {
+        "ContentScriptFilter": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/scripting/ContentScriptFilter",
+            "support": {
+              "chrome": {
+                "version_added": "88",
+                "notes": "Available for use in Manifest V3 only."
```

Comment:
I see this note is repeated a lot.  I think this would be better to put on the corresponding MDN article, rather than in notes, since this seems to be consistent for all browsers?

Reply:
If we could put a note on the feature itself, that would be ideal, otherwise, we really need to include these notes so that the information isn't disconnected.

Reply:
I do think that the information is helpful, but generally notes in BCD are meant to describe a difference in implementation that may affect (cross-)browser compatibility, such as variation from the spec.  This is tough because web extensions don't really have a "spec" to refer to in the first place, but the way I see it is: if this behavior is consistent across all browser engines, then it should be documented in mdn/content as a part of the feature rather than a BCD note to describe uncommon behavior.

Reply:
Yes, from some of the other changes I'm seeing that this is not an appropriate or sustainable approach. I'm going to revisit all the compatibility data changes when I've completed the core content. Hopefully early next week.